### PR TITLE
[107] - Printing Product Price with JavaScript - Fullstack

### PR DESCRIPTION
**Pull request by Marienoir**
## Task: `Printing Product Price with JavaScript `
- ` I Printed Product Price with JavaScript`

### CODE REVIEW

File: submissions/maryOkosun/javascript/productList.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	{
+		productName:"Tecno Cammon",
+		imageUrl:"https://www.google.com/imgres?imgurl=https%3A%2F%2Fimages.ctfassets.net%2Fd6skzop43my5%2F25VOo3VCXV8PpXXWtptIkK%2F0fa1442cbcae92076719ed9161b6574e%2Fhmd-home-phones-20191219.jpg%3Fq%3D50&imgrefurl=https%3A%2F%2Fwww.hmdglobal.com%2F&tbnid=r2XgcLf1fgqH6M&vet=12ahUKEwibopHhi4LtAhVN0RoKHS2TBgAQMygFegUIARCsAQ..i&docid=M-gE3rUyVsfVwM&w=1200&h=1200&q=phones%20images%20url&ved=2ahUKEwibopHhi4LtAhVN0RoKHS2TBgAQMygFegUIARCsAQ", 
+		productPrice:"$450"
+	}
+]
+for (var i = 0, l=productList.length; i< l; i++) {
+	console.log(productList[i].productName + " with " + productList[i].imageUrl + " is sold for " + productList[i].productPrice);
+}
+
```
Comment (adaugochi) [inline submissions/maryOkosun/javascript/productList.js:21]: Kindly remove all this white spaces here

**Review by adaugochi — CHANGES_REQUESTED** [review]
You are deleting other people images on this file.

File: submissions/maryOkosun/javascript/productList.js (hunk 1 above)
Comment (Marienoir) [reply submissions/maryOkosun/javascript/productList.js:10]: I am sorry. I do not even know how the deletion happened. I have fixed the issues with the whitespace. Kindly review. Thanks

**Review by adaugochi — COMMENTED** [review]
I can't approve this PR because it contains someone else's work that you are deleting
image
It might be kind of difficult to return back those images you deleted.

**Comment by adaugochi** [comment]
1. It either you close the PR and start afresh. (i.e checkout back to master, create a new working branch and redo your task, push and create a PR)

OR

2. You don't have to close the PR. Simply checkout to master, go to `fabunmiTolulope` folder and copy back those images that were deleted. Checkout back to your `js` branch and place the images back

**Pull request closed without merging**